src/index.scss

```diff
@@ -0,0 +1,50 @@
+/* Start Global Rules */
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+body {
+  font-family: "cairo", sans-serif;
+  background-color: #eee;
+}
+html {
+  scroll-behavior: smooth;
+}
+a {
+  text-decoration: none;
+}
+ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+.container {
+  padding-left: 15px;
+  padding-right: 15px;
+  margin-left: auto;
+  margin-right: auto;
+}
+/* Small */
+@media (min-width: 768px) {
+  .container {
+    width: 750px;
+  }
+}
+/* Medium */
+@media (min-width: 992px) {
+  .container {
+    width: 970px;
+  }
+}
+/* Large */
+@media (min-width: 1200px) {
+  .container {
+    width: 1170px;
```

Comment:
Good use of media queries however, it's preventing the article cards from stretching across the screen 
![Screenshot 2023-01-10 at 11 02 20](https://user-images.githubusercontent.com/91190841/211535550-1eff10b8-e0e8-4803-9657-e12f4564e078.png)